blog/src/User/Console/CreateCommand.php

```diff
@@ -41,25 +46,21 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $login = (string) $input->getArgument('login');
         $password = (string) $input->getArgument('password');
         $isAdmin = (bool) $input->getArgument('isAdmin');
-
-        $this->signupForm->load([
-            'login' => $login,
-            'password' => $password,
-            'passwordVerify' => $password,
-        ], '');
-
         try {
+            $this->formHydrator->populate(model: $this->signupForm, data: [
+                'login' => $login,
+                'password' => $password,
+                'passwordVerify' => $password,
+            ], scope: '');
             $user = $this->signupForm->signup();
         } catch (Throwable $t) {
             $io->error($t->getMessage() . ' ' . $t->getFile() . ' ' . $t->getLine());
-
             return $t->getCode() ?: ExitCode::UNSPECIFIED_ERROR;
         }
 
         if ($user === false) {
-            $errors = $this->signupForm->getFormErrors()->getFirstErrors();
-            array_walk($errors, fn ($error, $attribute) => $io->error("$attribute: $error"));
-
+            $errors = $this->signupForm->getValidationResult()?->getErrorMessagesIndexedByAttribute();
```

Comment:
Psalm Level 1 Testing: 
ERROR: TypeDoesNotContainNull - src/User/Console/CreateCommand.php:79:23 - Yiisoft\Validator\Result does not contain null (see https://psalm.dev/090)
            $errors = $this->signupForm->getValidationResult()?->getErrorMessagesIndexedByAttribute();

Reply:
Recommended: Remove the question mark in getValidationResult?